Tests/TokamakStaticHTMLTests/RenderingTests/Snapshotting+image.swift

```diff
@@ -0,0 +1,71 @@
+// Copyright 2021 Tokamak contributors
```

Comment:
nit: can this be renamed to `Snapshotting+Image.swift` for consistency with capitalization?